data/json/materials.json

```diff
@@ -36,6 +36,13 @@
     "bash_dmg_verb": "dented",
     "cut_dmg_verb": "scratched"
   },
+  {
+    "type": "material",
+    "ident": "acidchitin",
+    "copy-from": "chitin",
```

Comment:
It seems that ```copy-from``` doesn't work in materials definition:

![capture](https://user-images.githubusercontent.com/5421954/38464605-38308876-3b19-11e8-95c0-3213b30f6647.PNG)

Removing ```copy-from``` part by explicit definition of material fixes this issue, so I propose to use this as a quick patch.